### ISSUE TITLE
Doesn't support printf

### ISSUE THREAD

**Issue by l0b0** [jarv/cmdchallenge#121]
My entire session:

```
bash(0)> printf 'hello world'
Unknown Error :(
bash(☠️)> wat
Unknown Error :(
```

**Comment by jarv** [comment]
Hm, I don't see this. I assume you are seeing this consistently and that it never worked for you?
What browser are you using? Any chance you have a adblocker or browser extension enabled?

```
bash(0)> printf 'hello world'
hello world
# 👍 👍 👍  Correct!
# You have a new challenge!
# Print the current working directory.
# 
bash(0)> wat
wat: command not found
bash(127)>  
```

**Comment by l0b0** [comment]
Yes, I have a bunch of extensions enabled, including an ad blocker.

**Comment by jarv** [comment]
Hm, that could be the reason why. If you can try temporarily disabling them and trying again it would help to narrow down the problem. There are no ads and no external javascript on cmdchallenge.com if that makes you feel better about disabling them.

**Comment by l0b0** [comment]
I see in uMatrix that it does do an XHR to an "execute-api" host on AWS. Is that how you evaluate commands?

**Comment by jarv** [comment]
Yeah this is the likely culprit, it makes a cross-origin xhr cors request when you submit.

**Comment by l0b0** [comment]
Accidentally closed it. Do you think changing the error message to include more information if it can't reach the execute-api host would work?

**Comment by jarv** [comment]
Yes the error reporting is terrible here, I will try to improve it.
